f8a_worker/monkey_patch.py

```diff
@@ -0,0 +1,63 @@
+"""Functions for Monkey Patching in Selinon."""
+
+from datetime import datetime
+import json
+from selinon.errors import SelinonException
+from selinon import Dispatcher
+import os
+
+
+class NotABugFatalTaskError(SelinonException):
+    """An exception that is raised by task on fatal error - task will be not retried."""
+
+    def __init__(self, state):
+        """Make sure flow errors capture final state of the flow.
+
+        :param state: final flow details
+        """
+        super().__init__(json.dumps(state))
+
+    @property
+    def state(self):
+        """Get structured flow details."""
+        return json.loads(str(self))
+
+
+def _check_hung_task(self, flow_info):
+    """Remove tasks which are rotating in dispatcher for more than given time.
+
+    :param flow_info: information about the current flow
+    """
+    node_args = flow_info['node_args']
+    if node_args is not None and 'flow_start_time' in node_args \
+            and node_args['flow_start_time'] is not None:
+        flow_start_time = datetime.strptime(node_args['flow_start_time'], '%Y-%m-%d %H:%M:%S.%f')
+        current_time = datetime.now()
+        time_diff = current_time - flow_start_time
+        no_of_hours = time_diff.days * 24 + time_diff.seconds // 3600
+
+        node_args['no_of_hours'] = no_of_hours
+        dispatcher_time_out_in_hrs = int(os.environ.get('SQS_MSG_LIFETIME', '24'))
```

Comment:
Convert it as global package private constant and convert the hours to seconds to avoid repeated conversion during the condition testing.
```suggestion
     _SQS_MSG_LIFETIME_IN_SEC   = int(os.environ.get('SQS_MSG_LIFETIME', '24')) * 60 * 60
```